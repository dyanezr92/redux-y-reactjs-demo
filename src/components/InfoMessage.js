import React from 'react'
import PropTypes from 'prop-types'

const InfoMessage = ({ text }) => (
  <p className="alert alert-info">{text}</p>
)

InfoMessage.propTypes = {
  text: PropTypes.string.isRequired
}

export default InfoMessage
