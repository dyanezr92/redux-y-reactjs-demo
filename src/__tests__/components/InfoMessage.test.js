import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store'; // Smart components

import InfoMessage from '../../components/InfoMessage';

describe('<InfoMessage text="Alert text!"/>', () => {
  describe('render()', () => {
    test('renders the info message', () => {
      const wrapper = shallow(<InfoMessage text="Alert text!"/>);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
