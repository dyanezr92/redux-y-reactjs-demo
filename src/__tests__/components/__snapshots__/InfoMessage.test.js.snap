// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InfoMessage text="Alert text!"/> render() renders the info message 1`] = `
<p
  className="alert alert-info"
>
  Alert text!
</p>
`;
